### ISSUE TITLE
Wallaby fails to download and extract node.js

### ISSUE THREAD

**Issue by justinbarry** [wallabyjs/public#1711]
### Issue description or question
Wallaby fails to download and extract node. Further deps seem to download fine.

### Code editor or IDE name and version
WebStorm v2018.2 EAP

### OS name and version
OSX v10.13.5

![image](https://user-images.githubusercontent.com/328965/41450731-66b52be0-701e-11e8-98fe-f4ae8e4f01b0.png)

Stack trace:
```
java.lang.NoClassDefFoundError: com/intellij/util/io/tar/TarInputStream
	at com.wallaby.javascript.WallabyDependencies$d.a(Unknown Source)
	at com.wallaby.javascript.WallabyDependencies$d.a(Unknown Source)
	at com.wallaby.javascript.WallabyDependencies$d.b(Unknown Source)
	at com.wallaby.javascript.WallabyDependencies$d.a(Unknown Source)
	at com.wallaby.javascript.aR.run(Unknown Source)
	at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:247)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:736)
	at com.intellij.openapi.progress.impl.CoreProgressManager$5.run(CoreProgressManager.java:434)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:580)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:525)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:85)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
	at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:570)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:310)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: com.intellij.util.io.tar.TarInputStream PluginClassLoader[wallaby.js, 1.0.114] com.intellij.ide.plugins.cl.PluginClassLoader@51d0ca85
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:63)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 20 more
```

**Comment by ArtemGovorov** [comment]
Thanks for the report. Some IntelliJ platform API had been changed in the latest WS EAP. The issue is fixed and the fix is published in the latest plugin v1.0.115.

**Comment by justinbarry** [comment]
@ArtemGovorov Thank you for the swift response! I love the product.
